scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -637,7 +637,10 @@ def add_parse_arguments(parser = None):
         "-u",
         "--no-update",
         action="store_true",
-         help="Do not update the results of the last run of twister.")
+         help="Do not update the results of the last run. This option "
```

Comment:
```suggestion
         help="Do not update the resulting reports of the last run. This option "
```

Reply:
I do not think this is better.

Reply:
@pkunieck are you ok for #70173 with the new description and what 'results' are meant here as not updated ?

Reply:
I think results are just enough. It looks clear